packages/p2p/src/components/my-ads/reduce-order-time-banner.jsx

```diff
@@ -0,0 +1,53 @@
+import { observer } from 'mobx-react-lite';
+import React from 'react';
+import { useStores } from 'Stores';
+import { HintBox, Text } from '@deriv/components';
+import { localize } from 'Components/i18next';
+
+const ReduceOrderTimeBanner = () => {
+    const { general_store } = useStores();
+    const { order_timeout } = general_store;
+    // Convert the time in minutes to hours
+    const hours = Math.floor(order_timeout / 60);
+    const minutes = order_timeout % 60;
```

Comment:
Can we add this to the date-time.js file

Reply:
Will move it